app/views/houses/show.html.erb

```diff
@@ -37,11 +37,14 @@
 </table>
 
   <h1>Submit an offer</h1>
+  <% if current_user != @house.user %>
   <%= simple_form_for [@house, Offer.new] do |f| %>
     <%= f.input :price %>
     <%= f.submit "Submit offer", class: "btn btn-primary" %>
   <% end %>
-
+<% else %>
+  <p>You are the seller of this house, so you cannot submit an offer.</p>
```

Comment:
if i'm not wrong - there shouldn't be validation on sellers submitting offers because the offer form should not even be visible to sellers on your own associated houses